spec/spec_helper.rb

```diff
@@ -36,12 +37,15 @@ def database_id
 I18n.available_locales = [ :en, :nl ]
 
 RSpec.configure do |c|
+  c.raise_errors_for_deprecations!
+
+  c.before :all do
+    Mongoid.logger.level = Logger::INFO
+    Mongo::Logger.logger.level = Logger::INFO if Mongoid::Compatibility::Version.mongoid5?
+  end
+
   c.before(:each) do
     Mongoid.purge!
     Mongoid::IdentityMap.clear if defined?(Mongoid::IdentityMap)
   end
-
-  c.after(:suite) do
```

Comment:
I removed this cause I don't see why we need it, but it can be fixed with

```ruby
if Mongoid::Compatibility::Version.mongoid5?
  Mongoid.default_client.database.drop
else
  Mongoid.default_session.drop
end
```